src/styles/index.css

```diff
@@ -0,0 +1,72 @@
+@keyframes watch-animation-rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  color: #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1vmin;
+  border-style: dotted;
+  border-radius: 50%;
+}
+
+.stopwatch__hand {
+  bottom: 50%;
+  transform-origin: bottom;
+  position: absolute;
```

Comment:
The position of `.stopwatch__hand` is set to absolute, but the parent element `.stopwatch` does not have a `position: relative;`. This can cause unexpected behavior. Be sure to add `position: relative;` to the `.stopwatch` class.